configapi/api_inventory.go

```diff
@@ -1,231 +1,231 @@
+	if errGetMany != nil {
+		logger.DbLog.Errorln(errGetMany)
+		c.JSON(http.StatusInternalServerError, upfs)
+	}
+
+	for _, rawUpf := range rawUpfs {
+		var upfData configmodels.Upf
+		err := json.Unmarshal(configmodels.MapToByte(rawUpf), &upfData)
+		if err != nil {
+			logger.DbLog.Errorf("could not unmarshall UPF %v", rawUpf)
+		}
+		upfs = append(upfs, &upfData)
+	}
+	c.JSON(http.StatusOK, upfs)
 }
 
 func PostUpf(c *gin.Context) {
-    setInventoryCorsHeader(c)
-    if err := handlePostUpf(c); err == nil {
-        c.JSON(http.StatusOK, gin.H{})
-    } else {
-        c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
-    }
+	setInventoryCorsHeader(c)
+	if err := handlePostUpf(c); err == nil {
+		c.JSON(http.StatusOK, gin.H{})
+	} else {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+	}
 }
 
 func DeleteUpf(c *gin.Context) {
-    setInventoryCorsHeader(c)
-    if err := handleDeleteUpf(c); err == nil {
-        c.JSON(http.StatusOK, gin.H{})
-    } else {
-        c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
-    }
+	setInventoryCorsHeader(c)
+	if err := handleDeleteUpf(c); err == nil {
+		c.JSON(http.StatusOK, gin.H{})
+	} else {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+	}
 }
 
 func handlePostUpf(c *gin.Context) error {
-    var upfHostname string
-    var exists bool
-    if upfHostname, exists = c.Params.Get("upf-hostname"); !exists {
-        errorMessage := "post UPF request is missing upf-hostname"
-        configLog.Errorf(errorMessage)
-        return errors.New(errorMessage)
-    }
-    configLog.Infof("Received UPF %v", upfHostname)
-    var err error
-    var newUpf configmodels.Upf
-
-    allowHeader := strings.Split(c.GetHeader("Content-Type"), ";")
-    switch allowHeader[0] {
-    case "application/json":
-        err = c.ShouldBindJSON(&newUpf)
-    }
-    if err != nil {
-        configLog.Errorf("err %v", err)
-        return fmt.Errorf("failed to create UPF %v: %w", upfHostname, err)
-    }
-    if newUpf.Port == "" {
-        errorMessage := "post UPF request body is missing port"
-        configLog.Errorf(errorMessage)
-        return errors.New(errorMessage)
-    }
-    req := httpwrapper.NewRequest(c.Request, newUpf)
-    procReq := req.Body.(configmodels.Upf)
-    procReq.Hostname = upfHostname
-    msg := configmodels.ConfigMessage{
-        MsgType:     configmodels.Inventory,
-        MsgMethod:   configmodels.Post_op,
-        UpfHostname: upfHostname,
-        Upf: &procReq,
-    }
-    configChannel <- &msg
-    configLog.Infof("successfully added UPF [%v] to config channel.", upfHostname)
-    return nil
+	var upfHostname string
+	var exists bool
+	if upfHostname, exists = c.Params.Get("upf-hostname"); !exists {
+		errorMessage := "post UPF request is missing upf-hostname"
+		configLog.Errorf(errorMessage)
+		return errors.New(errorMessage)
+	}
+	configLog.Infof("Received UPF %v", upfHostname)
```

Comment:
Do you mind making this update as part of your PR?
```suggestion
	configLog.Infof("received UPF %v", upfHostname)
```